frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -206,7 +206,7 @@ const TableContainer = ({
         </div>
       )}
       <div className={`${baseClass}__header`}>
-        {data && data.length && !disableCount ? (
+        {data && !disableCount && (
```

Comment:
Is `data` always an array here?

Reply:
99.9% sure the data that goes into the TableContainer is always an array

Reply:
Ok. It would be a better check to have `!data?.length && !disableCount`

Reply:
Why?

Reply:
I think this was a recent EcmaScript addition that offers is a cleaner way to do the sort of nullish check that used to require the `array &&  array.length` check

Reply:
It's just the data is definitely an array so why do we need to check because data.length will always exist?

Reply:
The component catches !data before this doesn't it?

Reply:
`data.length` will be false if it equals zero. Check out this JS doc: https://developer.mozilla.org/en-US/docs/Glossary/Falsy